projects/the-internet/src/pages/infiniteScroll.js

```diff
@@ -1,17 +1,27 @@
 import { BASE_URL } from "../constants/misc";
 
 class infiniteScroll {
-  locators = {
-    profileInfo: ".figcaption",
-    viewProfileLink: "#content > div a",
-    pageNotFound: "body > h1"
-  };
+	locators = {
+		infiniteScrollText: ".jscroll-added",
+		placeholderText: "small",
+		infiniteScrollChild: ".jscroll-inner >",
+	};
 
-  visit() {
-    cy.visit(`${BASE_URL}/infinite_scroll`);
-  }
+	visit() {
+		cy.visit(`${BASE_URL}/infinite_scroll`);
+	}
 
+	getInfiniteText() {
+		return cy.get(this.locators.infiniteScrollText);
+	}
 
+	getPlaceholderText() {
+		return cy.get(this.locators.placeholderText);
+	}
+
+	getInfiniteScrollChild() {
```

Comment:
If you are already adding the child return as a method, consider passing in the index (ex. 'i') in the method itself and adding the rest of the selector (in this case it would be 'nth-child')